src/Sprite/ArtFile.h

```diff
@@ -14,10 +15,13 @@ namespace Stream {
 	class Writer;
 }
 
+// ArtFile stores sprites in 1 and 8 bit indexed bitmaps. 2 bit bitmaps represent shadows.
+// ArtFile may be packaged within a vol file and still register with Outpost 2.
```

Comment:
I think "1 bit bitmaps" (2 color) was intended here.

----

The phrasing "still register" isn't super clear.

Reply:
I'll rewrite the high level documentation and then merge.